README.md

```diff
@@ -62,7 +62,7 @@ use Spatie\Dns\Dns;
     ->getRecords('spatie.be');
 ```
 
-To filter the DNS record types you can use a string with the name of the record type, an array of names or one or multiple native php `DNS_XYZ` constants - for multiple you should use the `|` (pipe)  bit operator.
+To filter the DNS record types you can use a string with the name of the record type, an array of names or one or multiple native php `DNS_XYZ` constants - for multiple you should use the `|` (pipe) bit operator.
```

Comment:
Could you also add a code example? 👍

Reply:
https://github.com/spatie/dns/pull/60/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R52